apps/aarch64/filter1D/filter1D_cursor_forwarding.py

```diff
@@ -42,7 +42,6 @@ def filter1D(ow: size, kw: size, x: f32[ow + kw - 1], y: f32[ow], w: f32[kw]):
 # set memories & precision
 filter1D = set_memory(filter1D, "sum #1", Neon)
 filter1D = set_memory(filter1D, "xX4", Neon)
-filter1D = set_precision(filter1D, "xX4", "f32")
```

Comment:
Why was this removed?

Reply:
Since bind_expr automatically infers the type it is no longer necessary to set the precision.

Reply:
I see, nice!